src/aiq/tool/code_execution/test_code_execution_sandbox.sh

```diff
@@ -0,0 +1,373 @@
+#!/bin/bash
+
+# Test script for Code Execution Sandbox
```

Comment:
Is there a reason why bash + curl + jq is preferred over a simple python script?

In python you get tools like deindent (for better readability) and can more easily inspect the response.